syndeo-template/src/test/java/com/google/cloud/syndeo/perf/SyndeoLoadTestUtils.java

```diff
@@ -240,4 +252,77 @@ public static Row randomRowForSchema(
     }
     return rowBuilder.build();
   }
+
+  @AutoService(SchemaTransformProvider.class)
+  public static class GenerateDataSchemaTransformProvider
+      extends TypedSchemaTransformProvider<
+          GenerateDataSchemaTransformProvider.GenerateDataSchemaTransformConfiguration> {
+
+    @Override
+    public Class<GenerateDataSchemaTransformConfiguration> configurationClass() {
+      return GenerateDataSchemaTransformConfiguration.class;
+    }
+
+    @Override
+    public SchemaTransform from(GenerateDataSchemaTransformConfiguration configuration) {
+      return new SchemaTransform() {
+        @Override
+        public @UnknownKeyFor @NonNull @Initialized PTransform<
+                @UnknownKeyFor @NonNull @Initialized PCollectionRowTuple,
+                @UnknownKeyFor @NonNull @Initialized PCollectionRowTuple>
```

Comment:
Not sure you need any of these annotations (I know IntelliJ annoyingly adds them)